src/IdentityDecorator.php

```diff
@@ -111,6 +110,8 @@ public function __call(string $method, array $args): mixed
         }
         $call = [$this->identity, $method];
 
+        assert(is_callable($call), 'Method does not exist');
```

Comment:
I would prefer adding an inline ignore for the error instead of the `assert()`. PHP is gonna throw an error anyway if it's not a valid callable.